### PR TITLE
Add docker compose

### PR DESCRIPTION
**Pull request by linlinnn**
Fixed #9688
add ASF license for [PR](https://github.com/apache/pulsar/pull/9688) to pass CI

### CODE REVIEW

File: docker-compose/kitchen-sink/README.MD [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+- WebSocket (1)
+- Function (1)
+- Pulsar Manager (1)
+- SQL (1)
+
+## Motivation
+
+Most of what is available with Docker and Apache Pulsar uses the standalone version of Apache Pulsar, which is fantastic. But, users need to take care of all of the complexity of Apache Pulsar.
+
+This was created to create a Docker implementation that allows each individual part to be exposed to learn how it all fits together.
```
Comment (Huanli-Meng) [inline docker-compose/kitchen-sink/README.MD:44]:
```suggestion
This aims to create a Docker implementation that allows each individual part to be exposed to learn how it all fits together.
```

**Review by Huanli-Meng — COMMENTED** [review]
Leave my comments. PTAL

**Comment by sijie** [comment]
@Huanli-Meng Can you review it again?

**Review by Huanli-Meng — APPROVED** [review]
LGTM

**Comment by Huanli-Meng** [comment]
> @Huanli-Meng Can you review it again?

Done.

**Review by merlimat — APPROVED** [review]

**Pull request merged**